privacyidea/static/components/directives/controllers/directives.js

```diff
@@ -211,7 +211,11 @@ myApp.directive('assignToken', function($http, $rootScope, tokenUrl,
                 serial: "*" + $viewValue + "*"}
             }).then(function ($response) {
                 scope.newTokenObject.loadedSerials = $response.data.result.value.tokens.map(function (item) {
-                    return item.serial + " (" + item.tokentype + ")";
+                serial_string = item.serial + " [" + item.tokentype + "] "
```

Comment:
Would you change this back to rounded brackets, so we would keep the tokentype in rounded brackets and all additional information in square brackets.